website/addons/mendeley/views.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+
+from flask import request
+
+from framework.auth.decorators import must_be_logged_in
+
+from website.project.decorators import must_be_contributor_or_public
+from website.project.decorators import must_have_permission
+from website.project.decorators import must_not_be_registration
+from website.project.decorators import must_have_addon
+
+from .provider import MendeleyCitationsProvider
+
+
+@must_be_logged_in
+def list_mendeley_accounts_user(auth):
+    """ Returns the list of all of the current user's authorized Mendeley accounts """
+
+    provider = MendeleyCitationsProvider()
+    return provider.user_accounts(auth.user)
+
+
+@must_have_permission('write')
```

Comment:
Previously, the endpoint returned a list of folders associated with the user. Now, there is no need for it to be write.